asv_bench/benchmarks/index_object.py

```diff
@@ -199,3 +199,23 @@ def time_datetime_level_values_full(self):
 
     def time_datetime_level_values_sliced(self):
         self.mi[:10].values
+
+
+class IndexOps(object):
+    goal_time = 0.2
+
+    def setup(self):
+        N = 1000
+        idx = Index(range(10))
+        self.ridx = [idx] * N
+        self.iidx = [idx.astype(int)] * N
+        self.oidx = [idx.astype(str)] * N
```

Comment:
can can you post these results (commit before your recent fix of range index concat) to this commit